include/trieste/token.h

```diff
@@ -167,15 +167,23 @@ namespace trieste
     constexpr TokenDef::flag internal = 1 << 6;
   }
 
-  inline const auto Invalid = TokenDef("invalid");
+  // Built-in grouping
```

Comment:
While boyscouting here: Is there a reason for `Seq` and `NoChange` to not be `internal`?

Reply:
I think they should be.  When I added internal it was for a particular bit of code, but it makes sense to extend that here.